README.md

```diff
@@ -93,6 +93,12 @@ The default recipe includes the appropriate recipe based on the node's
 This recipe ensures the platform-specific configuration directory
 (`/etc/chef`) is created, and renders the configuration file
 (`push-jobs-client.rb`) using the `whitelist` attribute.
+Any environment variables can be set using `environment_variables`
+attribute with key value pairs.
+If you need to further customize the push-jobs-client.rb file,
+you can provide your own template in a wrapper cookbook and set the
```

Comment:
minor, could specify that you can provide your own _push-jobs-client.rb.erb_ template in a wrapper. Not sure if that level of detail is necessary here though.